meta-hovergames-distro/recipes-hovergames/images/imx-image-hovergames-demo.bb

```diff
@@ -399,6 +541,30 @@ fakeroot do_fix_ldconfig() {
 	set +x
 }
 
+
+fakeroot do_enable_bluetooth() {
+    set -x
+    
+    echo >> "${APTGET_CHROOT_DIR}/lib/systemd/system/hciattach.service" "[Unit] \n
+Description=Configure Bluetooth Modems connected by UART \n
+Before=bluetooth.service \n
+After=dev-serial1.device \n
+\n
+[Service] \n
+Type=forking \n
+ExecStartPre=-/usr/bin/hciattach /dev/ttymxc0 qualcomm -t120 115200 flow \n
+ExecStart=-/usr/bin/hciattach /dev/ttymxc0 qualcomm -t120 115200 flow \n
+ExecStopPost=-/usr/bin/hciattach /dev/ttymxc0 qualcomm -t120 115200 flow \n
```

Comment:
What's going on here? You issue the same command twice on startup? You issue the start command on stop?

Reply:
Indeed it looks a bit odd and thus probably wrong, it should be only started once, funny enough I don't see multiple instances of hicattach when running this service on the navq.